sunbird-cassandra-migration/cassandra-migration/src/main/resources/db/migration/cassandra/V1.88_cassandra.cql

```diff
@@ -0,0 +1 @@
+CREATE INDEX ON sunbird.shadow_user (userids);
```

Comment:
any cql chnages need to be added inside release-lms-2.5.0 , which course service is using.

Reply:
@manzarul done 
https://github.com/project-sunbird/sunbird-utils/pull/700